File: MPI/mpicheck/tests/functions/1.0/MPI_Cart_coords.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    int var_2;
    int var_3[2];
    int ret;
    /* calls */
    ret = MPI_Cart_coords(var_0, var_1, var_2, var_3);
    ret = PMPI_Cart_coords(var_0, var_1, var_2, var_3);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Cart_coords.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3(10)
       INTEGER var_4
        call mpi_cart_coords(var_0, var_1, var_2, var_3, var_4)
       call pmpi_cart_coords(var_0, var_1, var_2, var_3, var_4)
        end program main
    